scripts/zones/Jade_Sepulcher/mobs/Raubahn.lua

```diff
@@ -0,0 +1,81 @@
+-----------------------------------
+-- Area: Jade Sepulcher
+--  Mob: Raubahn
+-- BCNM: The Beast Within
+-----------------------------------
+mixins = { require("scripts/mixins/job_special") }
+require("scripts/globals/status")
+-----------------------------------
+local sepulcherID = require("scripts/zones/Jade_Sepulcher/IDs")
+-----------------------------------
+
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+
+    mob:addListener("ROAM_TICK", "RAUBAHN_RTICK", function(mob)
```

Comment:
Why are we using listeners instead of:
onMobRoam
onMobEngage
onMobDisengage
onMobFight

Reply:
special dialogs based on what is going on. Much like the maat mixin.

Reply:
But won't it behave identically if we code this just as we do any other mob?

Reply:
We use mixins when there are multiple versions of the same mob.